comixed-frontend/src/app/user/adaptors/authentication.adaptor.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('AuthenticationAdaptor', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [StoreModule.forRoot({ auth_state: reducer })],
-      providers: [AuthenticationAdaptor]
+      providers: [AuthenticationAdaptor],
```

Comment:
Still needs prettier run to clean up a few trailing commas like above.

(sorry if this feels like nitpicking, but I'm trying to keep things well-formatted)

To make it easier (and don't worry if it touches other files you didn't change, since I probably missed formatting a few files) you can do the following from the comixed-frontend directory:

```
prettier --write 'src/app/**/*.ts'
```

and commit all of the changes.

Reply:
It's ok I hope :) I like when the things are well-formatted too so it's not a problem.
I understand now why my prettier clean up was not good before. I used the latest version 2.0.5 and there are changes in the default behavior especially on trailingComma, arrowParens, and endOfLine options. (https://prettier.io/blog/2020/03/21/2.0.0.html)